README.adoc

```diff
@@ -18,7 +18,7 @@ You'll pick a Spring guide and import it into Spring Tool Suite. Then you can re
 == What you'll need
 
  - About 15 minutes
- - http://spring.io/tools/sts/all[Spring Tool Suite (STS)]
+ - https://spring.io/tools/sts/all[Spring Tool Suite (STS)]
```

Comment:
The target of this link doesn't exist anymore. I assume this is due the change of the STS Version.